components/wallet.tsx

```diff
@@ -1,181 +1,127 @@
-import { useWallet } from '@cosmos-kit/react';
+import { MouseEventHandler, useEffect, useMemo } from 'react'
```

Comment:
This file is pure create-cosmos-app afaik.
I would upstream any changes here to that. Calling @pyramation

Reply:
My goal here was to restyle the components here to use TailwindCSS. The original buttons are ChakraUI and include some data that is not necessary to this specific application. `create-cosmos-app` is the boilerplate for us to adapt to our needs :)